src/status_im2/navigation/view.cljs

```diff
@@ -67,7 +67,7 @@
         [rn/view {:style (wrapped-screen-style insets background-color)}
          [inactive]
          (if sheet?
-           [:f> bottom-sheet-screen/f-view component]
+           [bottom-sheet-screen/view {:content component}]
```

Comment:
could you pls elaborate on why
1) f removed
2) :content added

Reply:
1. As we need to use `with-theme`, I moved the `:f>` inside the component.

https://github.com/status-im/status-mobile/blob/b99e5cd6106c60d523d76dad06a08adb6cdb4f07/src/status_im2/common/bottom_sheet_screen/view.cljs#L83-L85

2. The theme is injected into the props (with the help of `with-theme`), and I believe having a map is better.  So, I added the `content` key as the component was received as `content` in the `bottom-sheet-screen` (before this PR).

In the existing `develop` branch:
https://github.com/status-im/status-mobile/blob/a6e6716311fb98dbb7d1b0cb72951757c8b6c2e1/src/status_im2/common/bottom_sheet_screen/view.cljs#L47

Changed to map:
https://github.com/status-im/status-mobile/blob/b99e5cd6106c60d523d76dad06a08adb6cdb4f07/src/status_im2/common/bottom_sheet_screen/view.cljs#L50